src/components/UpcomingSummits/index.jsx

```diff
@@ -0,0 +1,39 @@
+import React from 'react'
+
+import './styles.scss';
+
+import SummitCard from '../SummitCard';
+
+const UPCOMING_SUMMITS = [
+    {
+        key: 'asia-24',
+        background: 'img/summit-landing/cards/summit-asia.png',
+        date: 'September 2 & 3, 2024',
+        location: 'Suwon Convention Center, Suwon, South Korea',
+        notification: true
```

Comment:
use this as an object not a boolean
notification = {
title,
link
}
and if its present show the red notification with that data